packages/mollie-integration/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": ["../../.eslintrc.json"]
+}
```

Comment:
new line here.

Reply:
fixed

Reply:
There are some other files that are auto generated without end of line as well, can you please update them too 😄